### PR TITLE
complete diagnostic - Billy

### PR DESCRIPTION
**Pull request by byau**

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -7,27 +7,32 @@ Write your answers inside this file, where it's indicated by the comments.
...
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git clone git@github.com:byau/git-basics-diagnostic.git
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    cd git-basics-diagnostic
```
Comment (GA-MEB) [inline diagnostic.md:16]: I suppose that is technically true, but it's not what we were looking for. What should you do before starting to make your own changes?

File: diagnostic.md [hunk 2]
```diff
@@ -7,27 +7,32 @@ Write your answers inside this file, where it's indicated by the comments.
...
+
+    git checkout -b response // create a branch called 'response' and move into it
     ```
 
 4.  We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git diff
```
Comment (GA-MEB) [inline diagnostic.md:35]: Although this is correct, we haven't covered `git diff` yet officially; we were looking for `git status`.

File: diagnostic.md (hunk 1 above)
Comment (byau) [inline diagnostic.md:16]: create a branch and move into it ?

git checkout -b <branchname> ?  let me know if that’s what you are looking for?

**Comment by GA-MEB** [comment]
5/6 (-1 on Q2)

**Pull request closed without merging**